ci/terraform/oidc/variables.tf

```diff
@@ -686,6 +686,12 @@ variable "cmk_for_back_channel_logout_enabled" {
   description = "Feature flag which toggles whether the back channel logout queue is encrypted using CMK"
 }
 
+variable "back_channel_logout_call_timeout" {
+  default     = 3000
+  type        = number
+  description = "The HTTP Client connection timeout for requests to back channel logout URI (in milliseconds)."
+}
+
```

Comment:
I think we should make this a bit longer - the lambda can run for up to 30 seconds. AIS is one of our own services, so I think we'd expect that to come back quicker than a random RP service. How about 15 seconds?